packages/components/src/components/tick-progress/tick-progress.tsx

```diff
@@ -1,8 +1,19 @@
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
 
-const Tick = ({ is_on }) => {
+type TTick = {
+    is_on?: boolean;
+};
+
+type TTickProgress = {
+    className: string;
+    columns: number;
+    value: number;
+    rows: number;
+    size: number;
+};
+
+const Tick = ({ is_on }: TTick) => {
```

Comment:
if it only has one argument, why not typing it here instead of having `TTick`? :thinking:

Reply:
It can be changed as u suggested.